src/VecSim/vec_sim_tiered_index.h

```diff
@@ -88,7 +88,6 @@ class VecSimTieredIndex : public VecSimIndexInterface {
     }
 
     virtual VecSimIndexInfo info() const override;
-    virtual VecSimInfoIterator *infoIterator() const override;
```

Comment:
why?

Reply:
Because we have implementation-specific field ( swap jobs threshold)